main/settings.py

```diff
@@ -84,6 +84,7 @@
         "rest_framework.authentication.SessionAuthentication",
         "rest_framework_simplejwt.authentication.JWTAuthentication",
     ),
+    "TEST_REQUEST_DEFAULT_FORMAT": "json",
```

Comment:
I think this line is not needed because we can specify the format in the test itself. Any specific reasons for using this @decon-harsh because the tests are passing without it.

Reply:
Nothing , was just  following the django test docs given as the resources . I thought this is a healthy practice.

I will remove it.

Reply:
Removed